aggregation/src/Contracts/Signals.fs

```diff
@@ -17,6 +18,7 @@ module Signals =
             Longitude: decimal
             Timestamp: Instant
             ValidationResult: ValidationResult
+            ProviderId: Guid

```

Comment:
Note that this is a contract change. Some signals may be lost during deployment if we merge this PR as-is. An alternative would be to define `ProviderId` as `Guid option` and make it a `Guid` in a follow-up PR+deployment; this way the contract would be backwards compatible.